PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(14:44) when we look at apple stock right so we look at apple stock over the long term you know like this this this kind of meme of ultrasound money for ethereum you have this thing where it's mostly focused on whether it's inflating or deflating and what percentage right so for example apple stock is actually one of the most deflationary assets around it's it's got a 4.
+
+(15:05) 5 average annual deflation rate over the past 10 years they decreased their share account from 26 million to 16 million and that's of course one of the factors for why they did well on a per share basis but at the end of the day what it comes down to for because they're in equity it's the attractiveness of their ecosystem for customers right so it's not about how many shares they have because you have other ones like ibm that are also rapidly reducing their share count but because their ecosystems
+
+(15:30) unhealthy they're not really crewing that value to shares overall and so i think when it comes down to kind of what is trying to be money the thing that does almost nothing other than just be money is the best money and something that tries to do multiple things including proof of stake as one of the components inherently gives it less monetary premium in a proof of work system power is distributed between miners developers and individual nodes your ability to be a miner is based on your ability to put forth money and find low-cost
+
+(16:02) electricity rather than the original miners having an advantage and increasing their advantage over time new miners have a technical advantage over existing miners because they buy newer machines with more processing power if you if you bought a mining rig four years ago and i bought one right now i have it i have a bigger advantage or i can capture more coins that's rare that the new entrant has an advantage over the old entrant right so like you you came into it four years ago it's harder for you to capture and
+
+(16:35) mine the coins and it is for me and i think that what that does and when you look at proof of stake it's a little bit different if you had first access to the coins you now have an advantage over somebody who's now just coming in that's trying to capture them and employ them to the network and so from this system bitcoin mining businesses old and new are all in constant competition with each other this is opposite to proof of stake where there is no competition once you have ownership of the tokens you can keep
```

Comment:
Distinguere le frasi non del narratore `if you if you bought a mining rig four years ago and [...] who's now just coming in that's trying to capture them and employ them to the network`